hypha/apply/todo/models.py

```diff
@@ -28,6 +28,8 @@ class Task(models.Model):
     )
     related_object_id = models.PositiveIntegerField(blank=True, null=True)
     related_object = GenericForeignKey("related_content_type", "related_object_id")
+    # for manually added tasks
+    message = models.CharField(max_length=250, null=True, blank=True)
```

Comment:
I think comment edited case should be handled but a separate task message can provide us flexibility in changing the language of the task message, it might be helpful later or in another kind of tasks. And duplicity in terms of storage shouldn't be an issue as these tasks are temporary and will be removed regularly.